python/rascaline/tests/systems/ase.py

```diff
@@ -145,3 +146,54 @@ def test_no_pbc_cell():
     )
     with pytest.warns(Warning, match=message):
         AseSystem(atoms)
+
+
+def test_same_spherical_expansion():
+    system = ase.Atoms(
```

Comment:
I have the feeling that this cell is this to small to check for more then one unit cell?

Reply:
It's 3.6 in one direction, you should get pairs with a shift of 2 with a cutoff of 9